easybuild/cray_external_modules_metadata-20.10.cfg

```diff
@@ -15,6 +15,8 @@ name = LibSci
 
 [cray-netcdf]
 name = netCDF, netCDF-Fortran
+prefix = CRAY_NETCDF_DIR
+version = 4.7.4.2, 4.7.4.2
```

Comment:
why `4.7.4.2` twice ?

Reply:
Because you have two match the two packages listed in the field `name`